vmaas_sync/vmaas_sync.py

```diff
@@ -31,6 +31,7 @@
 VMAAS_REPOS_ENDPOINT = "%s/api/v1/repos" % VMAAS_HOST
 ENABLE_RE_EVALUATION = strtobool(os.getenv("ENABLE_RE_EVALUATION", "YES"))
 ENABLE_REPO_BASED_RE_EVALUATION = strtobool(os.getenv("ENABLE_REPO_BASED_RE_EVALUATION", "NO"))
+DEFAULT_PAGE_SIZE = 5000
```

Comment:
Could you change it please so the application reads it from environment variables?

Reply:
OK, I was thinking about creating function like this one in VMAAS:
https://github.com/RedHatInsights/vmaas/blob/4a8cacd7b8db34e21aa1818266174871f50d5dec/webapp/utils.py#L92-L95